assets/scss/components/_hscroll.scss

```diff
@@ -0,0 +1,84 @@
+/*doc
+---
+title: Horizontal Scroll
+name: hscroll
+category: Layout
+---
+Applies styles for horizontal scrolling lists.
+
+## Gradient `hscroll`
+Wrap the `hscroll` element with `hscrollGradientWrap` to add
+gradient overlays to both sides of the `hscroll`.
+
+## Variants
+
+A class of `hscroll` is __required__; the following classes are
+optional variants:
+
+Class                       | Description
+--------------------------- | ----------------------------------
+`atMedium_hscroll--unclip`  | Allows visible overflow (as inline block list) at medium & up
+`atLarge_hscroll--unclip`  | Allows visible overflow (as inline block list) at large & up
+
+*/
+.hscrollGradientWrap {
+	position: relative;
+	&:before, &:after {
+		content: '';
+		display: block;
+		position: absolute;
+		height: 100%;
+		width: $space;
+		top: 0;
+	}
+	&:before {
+		left: 0;
+		@include background-image(linear-gradient(to right, #fff, transparentize(#ffffff, 1)));
+	}
+	&:after {
+		right: 0;
+		@include background-image(linear-gradient(to left, #fff, transparentize(#ffffff, 1)));
+	}
+
+	.stripe--collection & {
+		&:before {
+			@include background-image(linear-gradient(to right, $C_collectionBG, transparentize($C_collectionBG, 1)));
+		}
+		&:after {
+			@include background-image(linear-gradient(to left, $C_collectionBG, transparentize($C_collectionBG, 1)));
+		}
+	}
+
+	.stripe--inverted {
+		&:before {
+			@include background-image(linear-gradient(to right, $C_contentBGInverted, transparentize($C_contentBGInverted, 1)));
+		}
+		&:after {
+			@include background-image(linear-gradient(to left, $C_contentBGInverted, transparentize($C_contentBGInverted, 1)));
+		}
+	}
+}
+
+.hscroll {
+	-webkit-overflow-scrolling: touch;
```

Comment:
should we use `-ms-overflow-style: -ms-autohiding-scrollbar;` or `-ms-overflow-style: none;` to hide the scroll bar?

More info here: https://developer.mozilla.org/en-US/docs/Web/CSS/-ms-overflow-style